### ISSUE TITLE
[Suggestion] Add ability to configure whether storage drawers retain contents when picked up (Or drop when broken like a chest)

### ISSUE THREAD

**Issue by paulbevz** [jaquadro/StorageDrawers#1073]
Or maybe there's a way to do this through datapacks ? I'm just not too keen on how

**Comment by IntegerLimit** [comment]
There is a config, present in the config file, called `keepContentsOnBreak`.

**Comment by paulbevz** [comment]
Thanks!! I didn't see it for some reason

**Comment by paulbevz** [comment]
> There is a config, present in the config file, called `keepContentsOnBreak`.

Okay so it's not present in the 1.19 version's config file so I'm not sure if the creator ported that feature ?

**Comment by IntegerLimit** [comment]
Yeah, he made it so that they would always drop with their items. I'm planning to make a clearing key, or something along those lines.

**Comment by kesumin** [comment]
@jaquadro are there plans for this in the future or does this feature already exist?

**Comment by Jack-McKalling** [comment]
This feature will not come back, since it crashes servers if the contents are spilled out into the world. This was an issue that was fixed with the removal of the toggle option and drawers always keeping their contents.
